src/SDKs/Automation/Management.Automation/Generated/ScheduleOperations.cs

```diff
@@ -220,7 +220,7 @@ internal ScheduleOperations(AutomationClient client)
             System.Net.HttpStatusCode _statusCode = _httpResponse.StatusCode;
             cancellationToken.ThrowIfCancellationRequested();
             string _responseContent = null;
-            if ((int)_statusCode != 200 && (int)_statusCode != 409)
+            if ((int)_statusCode != 200 && (int)_statusCode != 201 && (int)_statusCode != 409)
```

Comment:
@vrdmr Just to make sure, you are treating a `409` as a successful return code here?

Reply:
oh. Good catch. Fixing this in the spec.